Tasks/JavaToolInstallerV0/FileExtractor/JavaFilesExtractor.ts

```diff
@@ -92,29 +87,28 @@ export class JavaFilesExtractor {
                 taskLib.mkdirP(tempFolder);
 
                 // 1 extract compressed tar
-                this.sevenZipExtract(file, tempFolder);
+                await this.sevenZipExtract(file, tempFolder);
+
                 console.log(taskLib.loc('TempDir', tempFolder));
                 const tempTar = tempFolder + path.sep + taskLib.ls('-A', [tempFolder])[0]; // should be only one
                 console.log(taskLib.loc('DecompressedTempTar', file, tempTar));
                     
                 // 2 expand extracted tar
-                this.sevenZipExtract(tempTar, this.destinationFolder);
+                await this.sevenZipExtract(tempTar, this.destinationFolder);
 
                 // 3 cleanup temp folder
                 console.log(taskLib.loc('RemoveTempDir', tempFolder));
                 taskLib.rmRF(tempFolder);
-            } else if ('.zip' === fileEnding) {
```

Comment:
I think this will work fine, but why change this to use 7zip to extract .zip files on Windows?  Just curious

Reply:
I was trying to reduce the complexity of the task.  Previously the 7zip method was not working with zip files, but it has since been updated making this separate zip method unnecessary.